contents/teams/error-tracking/objectives.mdx

```diff
@@ -1,27 +1,56 @@
-* Make in-app searching more intuitive
-  * Add error tracking tab to taxonomic filter
-  * Add email to omnisearch
-  * Expose search via Max AI
-* Clearly call out issues with source maps and offer resolution steps
-* Support Docs team's goal to make error tracking docs amazing
\ No newline at end of file
+### Q4 2025 Objectives
+
+#### Go all in on AI (Driver: <TeamMember name="David Newell" />)
+* Implement issue autofixing
+  * Integrate with Tasks product for in-app experience
+  * Build terminal based experience within the CLI
+    * Add issue selection to MCP (e.g. by file, most impactful)
+    * Test generation to prevent regressions
+* Add exception capture auto instrumentation to the wizard
+* Integrations to capture exceptions for popular LLM / agent tools (e.g. Langchain)
+
+#### Ship mobile error tracking (Drivers: <TeamMember name="Oliver Browne" />, <TeamMember name="Manoel Aranda Neto" />)
+* [Sequencing TBD]
+* Support suppression rules via remote config
+* Add suppression of minified exceptions before cymbal processing
+* Build dashboard template including crash free sessions / users
+
+#### Research deep dive on customers (Driver: <TeamMember name="Cory Slater" />)
+* Understand push & pull factors with competitors
+* Understand the impression and usage of error tracking tools / workflows at AI-first companies
+* Decide whether customers see LLM issues as exceptions
+* Understand spend / contraction of existing customers
+* Every engineer to attend at least 2 customer interviews
+
+#### Increase trust in product (Driver: <TeamMember name="Hugues Pouillot" />)
+* Improve issue grouping
+  * Introduce a measure of grouping accuracy to base changes off
+  * Add fingerprint versioning to avoid duplicate issues when iterating
+  * Fix obvious mis-groupings based on minor stack trace variances
+* RFC for moving issues into ClickHouse to avoid dual DB querying
+* Reduce the number of stacktrace-less issues
+* Give better context when a recording is not found / source maps missing
+
+#### Improving context on issues  (Driver: <TeamMember name="Aleksander Błaszkiewicz" />)
+* Add breakdowns to issues (by release / person_id)
+* Show first affected release & relevant commit
+  * Stretch: Auto assign to commit author
+* Level up Node SDK (add contexts & tagging, $session_id propagation)
+* Integrate logs into session timeline
+* Capture code variables in Python
+
+#### Better controls for capturing relevant issues
+* Advanced snoozing (e.g. notify me if 100 more users affected)
+* Adaptive sampling configuration
+* Add billing limits to onboarding
+* Setup threshold based alerting / management (e.g. stop capturing spiking issues)
+
+#### Tracking spikes in usage
+* billing, threshold based alerting
+* update onboarding with more controls
+* Alert me if I get more than 1000 issues in an hour
+* Issues affecting more users
+
+#### Simplify setup
+- Rethink UI / UX of issue page
+- Simplify alerting / grouping / assignment
```

Comment:
We already have 5 meaty goals without these (that's 2 more than last time). I really think we have capacity for at most one of these goals. Totally happy if people want to remove parts of the above goals in favour of specific points from these but these felt less important than the first 5 goals

Reply:
Do we have something like side-goals noted somewhere?

Reply:
The goals should be challenging but feasible. So, if we know in advance that we don't have the capacity and aren't planning to hire, we should remove them.
Usually I keep an extra goal as a stretch goal, that is somewhat achievable as well, but not a must

Reply:
I saw one client pointing out that he loves Sentry's tidy layout (compared to ours) and I remember @daibhin mentioning that we should do something there. Maybe this is also worth including as a stretch goal?

Reply:
> feels like the 'Tracking spikes in usage' is important since i've heard a couple of times we spend quite some time refunding non-intentional spikes

Fair, point. It's probably less important than it was because we've actually implemented a lot of what we needed to prevent the runaway bills e.g. burst protection

I think we'll fold the most important sub points from each of these goals into some of those above